smack-integration-test/src/main/java/org/igniterealtime/smack/inttest/Configuration.java

```diff
@@ -550,6 +567,7 @@ public static Configuration newConfiguration(String[] testPackages)
 
         builder.setCompatibilityMode(properties.getProperty("compatibilityMode"));
 
+        builder.setTestRunResultProcessors(properties.getProperty("testRunResultProcessors"));
```

Comment:
Add the default-value logic here. That is, if getProperty() does not return something, set the default value.